doc/source/BuildingContainers.rst

```diff
@@ -0,0 +1,40 @@
+.. _BuildingContainers:
+
+
+Creating Containers
+********************
+
+=================
+Container recipes
+=================
+
+Container recipes are self-contained except that at container creation time (``spack stack create ctr ...``) package information from the common ``packages.yaml`` (or a manually provided version) is added to the ``packages:`` section, and specs are added in the section ``specs`` as defined in the template (default is an empty template with no specs.) Container recipes are defined in ``configs/containers``. Currently available container recipes can be found in ``configs/containers/``.
+
+**WORK IN PROGRESS**
```

Comment:
This section was moved from its original location in Environments.rst. Some of the meta comments were removed and consolidated with the paragraph above, but this is still a work in progress so I left the warning.